api.swagger.yaml

```diff
@@ -6850,6 +7014,152 @@ components:
           type: string
           example: '#FFFFFF'
       type: object
+    group-poll-schema:
+      allOf:
+        - type: object
+          properties:
+            uuid:
+              description: 'Invotra UUID of the group poll.'
+              type: string
+              format: uuid
+              example: '01234567-89ab-cdef-1234-56789abcdef0'
+        - $ref: '#/components/schemas/group-poll-schema-common'
+        - type: object
+          properties:
+            author:
+              description: 'The author of this group poll.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/user-trimmed-schema'
+            group:
+              description: 'The group for this group poll.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/group-schema'
+            options:
+              description: 'The options for this group poll.'
+              type: array
+              minItems: 2
+              maxItems: 26
+              items:
+                type: string
+              example: ['Pork', 'Beef', 'Mutton']
+            timestamp_created:
+              description: 'This is the timestamp (in seconds) of when the group poll was created.'
+              type: integer
+              example: 1510941596
+            timestamp_updated:
+              description: 'This is the timestamp (in seconds) of when the group poll was last updated.'
+              type: integer
+              example: 1510941696
+            comment_count:
+              description: 'Count of comments on the group poll.'
+              type: integer
+              example: 5
+            likes_count:
+              description: 'Count of likes on the group poll.'
+              type: integer
+              example: 5
+    group-poll-schema-update:
+      allOf:
+        - $ref: '#/components/schemas/group-poll-schema-common'
+        - type: object
+          properties:
+            author:
+              description: 'The author of this group poll.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the user.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            group:
+              description: 'The group for this group poll.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the group.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
```

Comment:
this allows moving an existing poll to a different group - have we confirmed that this is permitted?

Reply:
This should not be allowed

Reply:
took out